x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/routes.ts

```diff
@@ -27,3 +27,5 @@ export const OLD_SEARCH_INDEX_CRAWLER_DOMAIN_DETAIL_PATH = `${SEARCH_INDEX_PATH}
 
 export const ML_MANAGE_TRAINED_MODELS_PATH = '/app/ml/trained_models';
 export const ML_NOTIFICATIONS_PATH = '/app/ml/notifications';
+
+export const DEV_TOOLS_CONSOLE_PATH = '/app/dev_tools#/console';
```

Comment:
Did you try to use the `share` plugin url service for this? [example](https://github.com/elastic/kibana/blob/main/x-pack/plugins/serverless_search/public/application/components/try_in_console_button.tsx#L28) this does have to be done at runtime, and because the locator is async you might also need to put in in a `useMemo` depending on where you need to use it. 😔   

I'm not 100% sure on the guidelines on when to use locators vs constants for links to other parts of the app. But I _think_ it's best to use locator for dev tools since they can be disabled. and in that case you would not show the link at all. 

@sphilipse do you know when to use a locator vs constant?

Reply:
@demjened I'd be fine with updating this in a follow-up post-FF

Reply:
Sounds good, I'll address this in a separate PR.

Reply:
Theoretically we should always use a locator for cross-plugin links, but in practice approximately no one does this 🤷

Reply:
Filed a [tech debt ticket](https://github.com/elastic/enterprise-search-team/issues/4994) for this.